packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/SetUp/SetUp.tsx

```diff
@@ -154,66 +163,66 @@ export const SetUp: React.FC<SetUpProps> = ({
               }}
             >
               <AuctionDatePickerWrapper>
-                <AuctionDatePicker
-                  size="regular"
-                  label="Starting date"
-                  minDate={new Date()}
-                  disabled={!activeInputs.includes('auctionDuration')}
-                  items={[
-                    {
-                      value: new Date(),
-                      name: 'Right after listing',
-                    },
-                  ]}
-                  onChange={(value) => setAuctionDuration({ startDate: value })}
-                  value={auctionDate.startDate}
+                <Controller
+                  name="startDate"
+                  control={control}
+                  render={({ field: { onChange, value }, fieldState: { error } }) => (
+                    <AuctionDatePicker
+                      size="regular"
+                      label="Starting date"
+                      error={!!error}
+                      helperText={error?.message}
+                      minDate={new Date()}
+                      maxDate={(endDate?.type === 'date' && endDate.date) || undefined}
```

Comment:
There is a runtime parameter that dictates how far in the future the auction can start. You can access this value by `api.query.content.auctionStartsAtMaxDelta()` from the Polkadot API. This returns a number of blocks. We will need to take this into account here and also in validation. I'm aware this wasn't mentioned in the initial scope, so possibly let's do it as a followup to not drag this PR for too long